### ISSUE TITLE
Compile error on mac @ 06cc6b4316fac76910c4e837bfad5e47c24f75e8

### ISSUE THREAD

**Issue by testitem** [coder3101/cp-editor#129]
**Describe the bug**
Compile error on dev branch:

```
Undefined symbols for architecture x86_64:
  "TestCases::MAX_NUMBER_OF_TESTCASES", referenced from:
      TestCases::loadFromFile(QString const&) in TestCases.cpp.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [CPEditor.app/Contents/MacOS/CPEditor] Error 1
make[1]: *** [CMakeFiles/CPEditor.dir/all] Error 2
make: *** [all] Error 2
```

**To Reproduce**
Compile

**Expected behavior**
To compile

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by ouuan** [comment]
Is https://stackoverflow.com/questions/29588994/mac-error-undefined-symbols-for-architecture-x86-64 helpful?

**Comment by testitem** [comment]
seems that i'll need to compile with g++-9 instead of g++ lol

**Comment by ouuan** [comment]
> seems that i'll need to compile with g++-9 instead of g++ lol

So it's solved? You should close this issue then.

**Comment by testitem** [comment]
No, I still can't compile

**Comment by ouuan** [comment]
I really think you have to search on Google by yourself, I can only ask you questions which may not help you, and I don't have a Mac to test.

If you still can't find a solution try to delete `static` in this line:

https://github.com/coder3101/cp-editor/blob/9fa1fe4c6b4f8625dac9d68494a58c91b6f32ceb/include/TestCases.hpp#L118

But I don't think C++ will be compilable only on some OSs.

**Comment by testitem** [comment]
Will be fixed by #135

**Comment by ouuan** [comment]
Closed by a54a2cb.
